### ISSUE TITLE
[Feature]: use parallel downloading

### ISSUE THREAD

**Issue by zhouyiqing0304** [KSP-CKAN/CKAN#4138]
### Problem

enable parallel downloading to accelerate download speed

### Suggestion

CKAN's download speed can only reach 1MiB/s in some place.
parallel downloading is necessary

### Alternatives

aria2
https://github.com/aria2/aria2

**Comment by zhouyiqing0304** [comment]
@HebaruSan

**Comment by HebaruSan** [comment]
@zhouyiqing0304 can you explain what you mean here? CKAN already downloads in parallel.

**Comment by zhouyiqing0304** [comment]
More threads

**Comment by HebaruSan** [comment]
CKAN already downloads with multiple threads.
Closing as already done. Thanks for the suggestion.
